README.md

```diff
@@ -106,3 +106,7 @@ If [hubot-auth](https://github.com/hubot-scripts/hubot-auth) script is loaded,
 recommended you use the `forget` command instead of `drop`.
 
 `hubot: drop <factoid>`
+
+### FYI
```

Comment:
Let's just make this a "quoted string", i.e.

```md
> Learning a new factoid...
```

And remove the level 3 heading. Also it should probably be moved to where the create command is explained, and probably reword it to say "Creating a factoid requires..." rather than "Learning...".